recipes-core/dist-nilrt/dist-nilrt-efi-ab-gateway.bb

```diff
@@ -32,4 +38,5 @@ python do_package_ipk_append() {
 FILES_${PN}_x64 = "\
     /usr/share/nilrt/restore-mode-image-${MACHINE}.iso \
     /usr/share/nilrt/nilrt-install \
+    /usr/share/licenses/${PN}/MIT \
```

Comment:
Sorry to add another spanner to the pile, but I'm concerned that this might not be the correct way to distribute the license. I'm worried that declaring it here (as a generic owned-file of the package) is incorrect, since that would imply that other packages "own" the same file.

Is this how the other `-lic` subpackages work? What happens if you `opkg remove` a `-lic` package; does opkg remove the `MIT` license from the system, or is it intelligent enough to only remove it once *all owners* are uninstalled?

Reply:
I don't think I fully understand the concern. Which other packages are you referring to when you say _owners_? The installation location of the license is namespaced. Only dist-nilrt-efi-ab-gateway owns that license file.. What am I missing?

Reply:
You're not missing anything. I was missing the `${PN}` in the path.

You're right; that makes sense. Resolved.